cbdcconfig/config.go

```diff
@@ -88,9 +88,10 @@ type Config_AWS struct {
 
 type Config_GCP struct {
 	Enabled StringBool `yaml:"enabled"`
-	Region  string     `yaml:"region"`
 
-	_DefaultRegion string `yaml:"default-region"`
+	Zone string `yaml:"zone"`
```

Comment:
This change appears to be a breaking config change and would need to be done as part of the config versioning system that's built into dinocluster.  Separately from this though, I don't think selecting a specific zone is something that is meant to exist in the config file, but instead the region should be defined, and the specific zone would be auto-detected or auto-selected.

Reply:
noted